lucene-jbossmodules/pom.xml

```diff
@@ -17,65 +17,171 @@
+                    </execution>
+                    <execution>
+                        <id>gen-alias-luceneinternal</id>
+                        <phase>generate-resources</phase>
+                        <goals>
+                            <goal>copy-resources</goal>
+                        </goals>
+                        <configuration>
+                            <outputDirectory>${template.dir}/system/layers/base/org/apache/lucene/internal/${slot.simple.id}/</outputDirectory>
+                            <resources>
+                                <resource>
+                                    <directory>src/main/modules/aliases/lucene_internal</directory>
+                                    <filtering>true</filtering>
+                                </resource>
+                            </resources>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>gen-feature-pack-buildfile</id>
+                        <phase>generate-resources</phase>
+                        <goals>
+                            <goal>copy-resources</goal>
+                        </goals>
+                        <configuration>
+                            <outputDirectory>${featurepack.dir}</outputDirectory>
+                            <resources>
+                                <resource>
+                                    <directory>src/main/feature-pack-def</directory>
+                                    <filtering>true</filtering>
+                                </resource>
+                            </resources>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.wildfly.build</groupId>
+                <artifactId>wildfly-feature-pack-build-maven-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>feature-pack-build</id>
+                        <goals>
+                            <goal>build</goal>
+                        </goals>
+                        <phase>compile</phase>
+                        <configuration>
+                            <config-file>feature-pack-template.xml</config-file>
+                            <config-dir>${featurepack.dir}</config-dir>
+                            <resourcesDir>/</resourcesDir>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
             <plugin>
                 <artifactId>maven-assembly-plugin</artifactId>
                 <configuration>
                     <finalName>${project.artifactId}-${project.version}</finalName>
                     <descriptors>
-                        <descriptor>${basedir}/src/main/assembly/dist.xml</descriptor>
+                        <descriptor>${basedir}/assembly.xml</descriptor>
```

Comment:
Any particular reason to move the assembly there? We already put the assembly configuration in `src/main/assembly` in about every project, be it when producing WildFly modules or distribution ZIPs... If possible I'd rather keep it all consistent.

Reply:
I moved it to be consistent with the WildFly projects building feature packs ;)

I also think it's cleaner; the `src/main` path is meant for source code.